x-pack/plugins/ingest_pipelines/public/application/sections/pipelines_list/main.tsx

```diff
@@ -179,9 +179,22 @@ export const PipelinesList: React.FunctionComponent<RouteComponentProps> = ({
             <EuiCallOut
               iconType="faceSad"
               color="danger"
-              title={i18n.translate('xpack.ingestPipelines.list.loadErrorTitle', {
-                defaultMessage: 'Cannot load pipelines, please refresh the page to try again.',
-              })}
+              title={
+                <FormattedMessage
+                  id="xpack.ingestPipelines.list.loadErrorTitle"
+                  defaultMessage="Unable to load pipelines. Try {reloadLink} the page."
```

Comment:
The way this text is phrased, it seems like clicking the reload link will reload the page, which I think of as a browser page refresh. But clicking this link will actually just resend the request to load the pipelines. Can we align text with action more closely by phrasing this as "Try reloading them"?

Reply:
Good point! I went with Gail's suggestion: "Try again".